Improve DSP Tooltip

evgengoldwar opened the pull request:
### **Dyson Sphere Module Launch Site**
![image](https://github.com/user-attachments/assets/05e51372-086c-471b-bb8a-a546d0fce86a)

### **Dyson Sphere Ray Receiving Station**
![image](https://github.com/user-attachments/assets/cff898dc-6442-4ca2-886a-41d0c757285e)

Taskeren commented:
Why do we need UUID in the tooltip?

evgengoldwar commented:
> Why do we need UUID in the tooltip? And there are some entries missing space between colons and values, please fix it. Also the remove the spaces between entry names and colons.

I will fix the rest, but about the UUID it was originally added so I did not remove it if necessary I can remove it?

Taskeren commented:
> > Why do we need UUID in the tooltip? And there are some entries missing space between colons and values, please fix it. Also the remove the spaces between entry names and colons.
> 
> I will fix the rest, but about the UUID it was originally added so I did not remove it if necessary I can remove it?

Idk, if so, keeping it is fine.

evgengoldwar commented:
> > > Why do we need UUID in the tooltip? And there are some entries missing space between colons and values, please fix it. Also the remove the spaces between entry names and colons.
> > 
> > 
> > I will fix the rest, but about the UUID it was originally added so I did not remove it if necessary I can remove it?
> 
> Idk, if so, keeping it is fine.

Fixed and updated screenshots
